lib/db.gs.js

```diff
@@ -14,24 +13,24 @@ if (api.gs) {
   // Example: { alias: 'gs', url: 'gs://metarhia.com/', storage: 'dbName' }
   api.db.gs.open = (database, callback) => {
     const application = database.application;
-    application.on('databasesOpened', () => {
-      const config = database.config;
-      if (config.path) config.provider = 'fs';
-      const db = application.databases[config.storage];
-      if (db) {
-        config.gs = api.gs;
-        config.db = db.connection;
-        config.provider = db.schema;
-        database.connection = api.gs;
-      }
-      api.gs.open(config, err => {
+    const config = database.config;
+    const provider = api.gs('pg', config);
+    api.gs.provider = provider;
```

Comment:
Does this mean that provider instance's methods will be accessible via `api.gs.provider.methodName()`? Or is there a shorthand available, maybe something similar to `gs.methodName()`?

Reply:
In applications you can use database alias (from configs)